modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/utils/SpringThinClientConfig.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+    <bean id="igniteClient" class="org.apache.ignite.internal.ducktest.utils.IgniteClientSpringBean">
+        <property name="clientConfiguration">
+            <bean class="org.apache.ignite.configuration.ClientConfiguration">
+                <property name="addresses">
+                    <list>
+                        <value>ducker01:10800</value>
+                        <value>ducker02:10800</value>
+                        <value>ducker03:10800</value>
+                        <value>ducker04:10800</value>
+                        <value>ducker05:10800</value>
+                        <value>ducker06:10800</value>
+                        <value>ducker07:10800</value>
+                        <value>ducker08:10800</value>
+                        <value>ducker09:10800</value>
+                        <value>ducker10:10800</value>
```

Comment:
will this work or read the environment?
looks like some hardcode or autogenerated code.